Cargo.toml

```diff
@@ -11,12 +11,7 @@ repository = "https://github.com/projectfluent/fluent-langneg-rs"
 readme = "README.md"
 categories = ["internationalization", "localization"]
 edition = "2021"
-include = [
-	"src/**/*",
-	"benches/*.rs",
-	"Cargo.toml",
-	"README.md"
-]
+include = ["src/**/*", "benches/*.rs", "Cargo.toml", "README.md"]
```

Comment:
Please don't include this style change here. It's not related to the PR and will be duplicated by other PRs normalizing meta data as soon as the safe-harbor release goes out. Also pattern-per-line here is easier to diff across crates. The tab bit will get fixed.

Reply:
Rebase has been applied here too